migrations/initial_0000.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS Customers(ID INT AUTO_INCREMENT PRIMARY KEY, email VARCHAR(255), state VARCHAR(2));
```

Comment:
Not sure why `addProductSku_0001.sql` is empty?

Reply:
I had to remove the command as it was still erroring and not letting me run the cli and forgot to add it back when I added in the lastRanID parameter, I'll add it back in the next commit

Reply:
gotcha 👍